core_backend/app/contents/models.py

```diff
@@ -178,6 +180,24 @@ async def update_content_in_db(
         return content_db
 
 
```

Comment:
Is it a temporar thing or do we want to call the db and update all contents that appeared in the top 5? I think this is one more use case for `redis`

Reply:
Not temporary. We want to keep track of all content being shared back so we can show "most frequently shared content".
This may not be the final way we do it (e.g. we cant track WHEN the last comment was shared) but is sufficient for now.